tests/test_cursor.py

```diff
@@ -442,6 +442,12 @@ def test_workgroup(self, cursor):
         cursor.execute('SELECT * FROM one_row')
         self.assertEqual(cursor.work_group, WORK_GROUP)
 
+    def test_no_s3_staging_dir(self):
```

Comment:
Please use the `with_cursor` decorator.
```
@with_cursor(work_group=WORK_GROUP)
def test_no_s3_staging_dir(self):
    cursor = conn.cursor(s3_staging_dir=None)
    cursor.execute('SELECT * FROM one_row')
    self.assertNotEqual(cursor.output_location, None)
```

Reply:
https://github.com/laughingman7743/PyAthena/commit/5c660bc3a46d4ee4f1af16eb76ddb0cf70d13760 🤔

Reply:
A conn variable created in the context of with_cursor cannot be referenced from test_no_s3_staging_dir.

Can I rewrite the private variables of the cursor directly?

```python
@with_cursor(work_group=WORK_GROUP)
def test_no_s3_staging_dir(self, cursor):
    cursor._s3_staging_dir = None
    cursor.execute('SELECT * FROM one_row')
    self.assertNotEqual(cursor.output_location, None)
```

Reply:
👌

Reply:
Fixed.